lib/std/sort.zig

```diff
@@ -34,18 +34,23 @@ pub fn insertion(
     insertionContext(0, items.len, Context{ .items = items, .sub_ctx = context });
 }
 
-/// Stable in-place sort. O(n) best case, O(pow(n, 2)) worst case.
-/// O(1) memory (no allocator required).
-/// `context` must have methods `swap` and `lessThan`,
-/// which each take 2 `usize` parameters indicating the index of an item.
-/// Sorts in ascending order with respect to `lessThan`.
-pub fn insertionContext(a: usize, b: usize, context: anytype) void {
-    assert(a <= b);
-
-    var i = a + 1;
-    while (i < b) : (i += 1) {
+/// Stable in-place sort.
+/// `context` must have methods `swap` and `lessThan`, with the following signatures:
+/// ```
+/// pub fn swap(self: Context, lhs: usize, rhs: usize) void
+/// pub fn lessThan(self: Context, lhs: usize, rhs: usize) bool
+/// ```
+/// `lhs` and `rhs` represent the indexes of the elements.
+/// Sorts from [low..high), excluding high, in ascending order with respect to `lessThan`.
+/// Computational complexity: O(n) best case, O(pow(n, 2)) worst case.
```

Comment:
```suggestion
/// Computational complexity: O(n) best case, O(n^2) worst case.
```

minor, but I'd prefer `O(n^2)` like it's done in other files:

```zig
lib/std/zig/AstGen.zig
31:/// would be O(N^2).

lib/std/sort/block.zig
784:    // kind of like how the O(n^2) insertion sort is used in some places

lib/std/crypto/codecs/asn1/der/ArrayListReverse.zig
2://! in O(n) instead of O(n^depth) at the cost of extra buffering.
```